### PR TITLE
Add endpoint and template for label badge

### PR DESCRIPTION
**Pull request by vidya-ram**

### CODE REVIEW

File: funnel/templates/label_badge.html.jinja2 [hunk 1]
```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<head>
+  <script src='https://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js'></script>
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
+  <style>
+    @page {
+      size: 62mm 68mm;  /* width height */
```
Comment (jace) [inline funnel/templates/label_badge.html.jinja2]: I think this should come from page config that's stored in the project. It's okay for now, but file a ticket for configurable badge layouts.

File: funnel/views/participant.py [hunk 2]
```diff
@@ -173,6 +173,21 @@ def participant_badge(profile, project, participant):
...
 
+@app.route('/<profile>/<project>/participant/<participant_id>/label_badge')
+@funnelapp.route('/<project>/participant/<participant_id>/badge', subdomain='<profile>')
+@lastuser.requires_login
+@load_models(
+    (Profile, {'name': 'profile'}, 'g.profile'),
+    ((Project, ProjectRedirect), {'name': 'project', 'profile': 'profile'}, 'project'),
+    (Participant, {'id': 'participant_id'}, 'participant'),
+    permission='checkin_event',
+)
```
Comment (jace) [inline funnel/views/participant.py]: @iambibhas Use of `load_models` should be deprecated in favour of `ModelView`. Applies to the other views in this file as well.

File: funnel/views/participant.py [hunk 3]
```diff
@@ -276,3 +291,27 @@ def event_badges(profile, project, event):
...
+@funnelapp.route('/<project>/event/<name>/badges', subdomain='<profile>')
+@lastuser.requires_login
+@load_models(
+    (Profile, {'name': 'profile'}, 'g.profile'),
+    ((Project, ProjectRedirect), {'name': 'project', 'profile': 'profile'}, 'project'),
+    (Event, {'name': 'name', 'project': 'project'}, 'event'),
+    permission='checkin_event',
+)
+def event_label_badges(profile, project, event):
+    badge_printed = True if request.args.get('badge_printed') == 't' else False
```
Comment (jace) [inline funnel/views/participant.py]: `badge_printed = getbool(request.args.get('badge_printed'))`

Comment (jace) [reply funnel/views/participant.py]: Import `getbool` from `coaster.utils`.

File: funnel/views/participant.py [hunk 4]
```diff
@@ -276,3 +291,27 @@ def event_badges(profile, project, event):
...
+    participants = (
+        Participant.query.join(Attendee)
+        .filter(Attendee.event_id == event.id)
```
Comment (jace) [inline funnel/views/participant.py]: `.filter(Attendee.event == event)` (always use relationships as they are less accident-prone than using id numbers.)

File: funnel/views/participant.py (hunk 2 above)
Comment (iambibhas) [reply funnel/views/participant.py]: Keeping this for a separate PR. All the views on this file need to be moved to ModelView.
